integration-test/src/main/java/methods/ScreenShotMethods.java

```diff
@@ -0,0 +1,46 @@
+package methods;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+
+import env.BaseTest;
+
+public class ScreenShotMethods implements BaseTest {
+    /** Method to take screen shot and save in ./Screenshots folder */
+    public void takeScreenShot() throws IOException {
+        final File scrFile = ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);
+
+        final DateFormat dateFormat = new SimpleDateFormat("yyyyMMddHHmmss");
+        final Calendar cal = Calendar.getInstance();
+        System.out.println(dateFormat.format(cal.getTime()));
+
+        final String scrFilepath = scrFile.getAbsolutePath();
+        System.out.println("scrFilepath: " + scrFilepath);
+
+        final File currentDirFile = new File("Screenshots");
+        final String path = currentDirFile.getAbsolutePath();
+        System.out.println("path: " + path + "+++");
+
+        System.out.println("****\n" + path + "\\screenshot" + dateFormat.format(cal.getTime()) + ".png");
+
+        FileUtils.copyFile(scrFile, new File(path + "\\screenshot" + dateFormat.format(cal.getTime()) + ".png"));
```

Comment:
Using file path builder/helper could help making it works on linux/windows, I am not sure.

Reply:
+1